src/boards/nice_nano/board.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 Nick Winans
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#ifndef _NICENANO_H
+#define _NICENANO_H
+
+#define _PINNUM(port, pin)    ((port)*32 + (pin))
+
+/*------------------------------------------------------------------*/
+/* LED
+ *------------------------------------------------------------------*/
+#define LEDS_NUMBER       1
+#define LED_PRIMARY_PIN   _PINNUM(0, 15) // Blue
+#define LED_STATE_ON      1
+
+/*------------------------------------------------------------------*/
+/* BUTTON
+ *------------------------------------------------------------------*/
+#define BUTTONS_NUMBER    2  // none connected at all
+#define BUTTON_1          _PINNUM(0, 18)  // unusable: RESET
+#define BUTTON_2          _PINNUM(0, 19)  // no connection
+#define BUTTON_PULL       NRF_GPIO_PIN_PULLUP
+
+//--------------------------------------------------------------------+
+// BLE OTA
+//--------------------------------------------------------------------+
+#define BLEDIS_MANUFACTURER  "Nice Keyboards"
+#define BLEDIS_MODEL         "nice!nano"
+
+//--------------------------------------------------------------------+
+// USB
+//--------------------------------------------------------------------+
+#define USB_DESC_VID           0x239A
+#define USB_DESC_UF2_PID       0x0029
```

Comment:
the VID/PID is already used and we don't allow to reuse it anymore. If your board can also run circuitpython, you could apply for a set of VID/PID from Adafruit.

Reply:
I'm not familiar with Circuit Python. Is there any reason I wouldn't be able to use Circuit Python on this nRF52840 board? If not, could you direct me where to add support and apply for a VID/PID from Adafruit? Thanks!

Reply:
join circuitpyhon discords channel and ask for a donation of VID/PID to see if you are eligible for one.